README.md

```diff
@@ -104,6 +104,10 @@ Inject Datadog trace id into logs for correlation. Defaults to `true`.
 
 Set to `true` to merge the X-Ray trace and the Datadog trace, when using both the X-Ray and Datadog tracing. Defaults to `false`.
 
+### DD_TRACE_DISABLED_PLUGINS
```

Comment:
Since this is not directly supported by `datadog-lambda`, Should we simply add a paragraph about the environment variables that are honored by `dd-trace` and a link to that list? https://docs.datadoghq.com/tracing/setup/nodejs/#configuration

Reply:
Good call - removed this and added a paragraph in `Trace` https://github.com/DataDog/datadog-lambda-js/pull/103/commits/d93ca224074798025631ca4df027e3179b7dc3a2